### ISSUE TITLE
怎么样从一个会话直接传输文件到另一个会话

### ISSUE THREAD

**Issue by PowerDi** [kingToolbox/WindTerm#845]
其实是类似于xftp的一个特性。可以不需要先拷贝到本机设备再传输到对端，所以一直没法完全替代xftp，与xshell等终端相比太甜品了。

**Comment by kingToolbox** [comment]
Does `xftp` support transferring files directly from one host to another?

I have planned to implement this feature before, but I haven't had the time yet. As far as I know, Only `FXP (File eXchange Protocol)`, which is disabled on majority of FTP servers, lets you copy files from one FTP-server to another using an FXP-client. And I don't think the `sftp` protocol supports the direct transfer of files between servers.

Usually, files are transferred between two servers by executing the `scp` or `sftp` command after logging in to one server. This operation is a bit cumbersome, especially when transferring large numbers of files. I hope to improve this operation so that users can easily transfer files between the two servers, of course, the files still need to be downloaded from one server to the local and then uploaded to the other server. But all this happens in memory, and it's uploading while downloading, similar to streaming the files.

**Comment by PowerDi** [comment]
Yes, u are right. The way that `xftp` must first download the files to the install path and then pass them to another client. So if the steps are broken down, the files would still stay in the disk.
